### ISSUE TITLE
Improve our docs using yearn docs

### ISSUE THREAD

**Issue by Uxio0** [safe-global/safe-infrastructure#53]
-  https://github.com/numan/yearn-gnosis-safe

**Comment by gzliudan** [comment]
This repo is based on AWS. Hope there is a new document just based on Ubuntu, so it can be used with all cloud providers.

**Comment by Uxio0** [comment]
> This repo is based on AWS. Hope there is a new document just based on Ubuntu, so it can be used with all cloud providers.

Our current docker compose setup can be used with every VPS

**Comment by gzliudan** [comment]
Yes, also hope there is a document for production which can be used with every VPS and not use docker.

**Comment by comverser** [comment]
Hello @Uxio0,
- Does [Running our services locally](https://github.com/safe-global/safe-infrastructure/blob/main/docs/running_locally.md) can be used for production?
- Can this method be used to set up an EC2 instance in AWS?

**Comment by Uxio0** [comment]
> Does [Running our services locally](https://github.com/safe-global/safe-infrastructure/blob/main/docs/running_locally.md) can be used for production? If so, how to remove the port number 8080 from the URL http://localhost:8080/?

You need to update the docker compose file to remove it and also make adjudments like using a reverse proxy with https

> Can this method be used to set up an EC2 instance in AWS for public deployment? 

This repository is only an example on how our stack runs together, it's only meant to be taken as an example
